README.md

```diff
@@ -1,89 +1,361 @@
+project. This includes JavaScript files that would be available to your compiled project.
+
+## Quickstart
+
+After installing Neutrino and the TaskCluster preset, add a new directory named `src` in the root of the project, with
+a single JS file named `index.js` in it.
+
+```bash
+❯ mkdir src && touch src/index.js
+```
+
+Edit your `src/index.js` file with the following:
+
+```js
+import { createServer } from 'http';
+
+const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+const port = process.env.PORT || 3000;
+
+createServer(async (req, res) => {
+  await delay(500);
+  console.log('Request!');
+  res.end('hi!');
+})
+.listen(port, () => console.log(`Server running on port ${port}`));
+```
+
+Now edit your project's package.json to add commands for starting and building the application.
+
+```json
+{
+  "scripts": {
+    "start": "neutrino start --use neutrino-preset-taskcluster",
+    "build": "neutrino build --use neutrino-preset-taskcluster"
+  }
+}
+```
+
+Start the app, then either open a browser to http://localhost:3000 or use curl from another terminal window:
+
+#### Yarn
+
+```bash
+❯ yarn start
+Server running on port 3000
+```
+
+```bash
+❯ curl http://localhost:3000
+hi!
+```
+
+#### npm
+
+```bash
+❯ npm start
+Server running on port 3000
+```
+
+```bash
+❯ curl http://localhost:3000
+hi!
+```
+
+## Building
+
+`neutrino-preset-taskcluster` builds assets to the `build` directory by default when running `neutrino build`. Using the
+quick start example above as a reference:
+
+```bash
+❯ yarn build
+clean-webpack-plugin: /taskcluster/build has been removed.
+Build completed in 0.419s
+
+Hash: 89e4fb250fc535920ba4
+Version: webpack 2.5.1
+Time: 424ms
+       Asset     Size  Chunks             Chunk Names
+    index.js  4.29 kB       0  [emitted]  index
+index.js.map  3.73 kB       0  [emitted]  index
+✨  Done in 1.51s.
+```
+
+You can either serve or deploy the contents of this `build` directory as a Node.js module, server, or tool. For Node.js
+this usually means adding a `main` property to package.json pointing to the built entry point. Also when publishing your
+project to npm, consider excluding your `src` directory by using the `files` property to whitelist `build`,
+or via `.npmignore` to blacklist `src`.
 
 ```json
 {
   "main": "build/index.js",
   "files": [
     "build"
-  ],
-  "config": {
-    "preset": "neutrino-preset-taskcluster"
-  },
-  "scripts": {
-    "build": "neutrino build",
-    "test": "neutrino test"
-  }
+  ]
 }
 ```
 
-Add your source code to `src/`, which is compiled using Babel's es2015 preset.
+_Note: While this preset works well for many types of Node.js applications, it's important to make the distinction
+between applications and libraries. This preset will not work optimally out of the box for creating distributable
+libraries, and will take a little extra customization to make them suitable for that purpose._
```

Comment:
I assume this is numerically most uses of this preset, since we really only have one application?